common/lib/xmodule/xmodule/modulestore/tests/test_contentstore.py

```diff
@@ -199,6 +199,18 @@ def test_copy_assets(self, deprecated):
         __, count = self.contentstore.get_all_content_for_course(dest_course)
         self.assertEqual(count, len(self.course1_files))
 
+    @ddt.data(True, False)
+    def test_copy_assets_with_duplicates(self, deprecated):
```

Comment:
I am not able to understand at which level in test `FileExists` is raised and handed?

Reply:
I am calling `copy_all_course_assets` in the test which raises the `FileExists`. There are two courses both have 3 assets each and one of the assets is common so when copying one course's assets into the other it raises already exists error. The new course has a total of 5 assets since one asset was common

Reply:
got it. thanks.